### PR TITLE
Update to Gear Sheet

### PR DESCRIPTION
**Pull request by JFranklin3000**
Update to Gear sheet to remove unused fields add a Type field and added Effects.

### CODE REVIEW

File: template.json [hunk 1]
```diff
@@ -441,14 +441,7 @@
       "templates": [
         "itemDescription"
       ],
-      "effectArea": null,
-      "effectShape": null,
-      "lightRange": {
-        "bright": null,
-        "dim": null
-      },
-      "quantity": 1,
```
Comment (phildominguez) [inline template.json:450]: I feel like quantity could still be good to have. If we end up removing it, the chip still shows up atm:
image

File: templates/item/item-gear-sheet.hbs [hunk 2]
```diff
@@ -3,45 +3,35 @@
   {{!-- Item Sheet Header --}}
   {{> "systems/essence20/templates/item/parts/item-header.hbs"}}
 
+    {{!-- Tabs --}}
```
Comment (phildominguez) [inline templates/item/item-gear-sheet.hbs:6]:
```suggestion
{{!-- Tabs --}}
```

File: templates/item/item-gear-sheet.hbs [hunk 3]
```diff
@@ -3,45 +3,35 @@
...
 
-    {{!-- Description Section --}}
-    {{> "systems/essence20/templates/item/parts/item-description.hbs"}}
+      {{!-- Description Section --}}
+      {{> "systems/essence20/templates/item/parts/item-description.hbs"}}
+    </div>
+    <div class="tab effects flexcol" data-group="primary" data-tab="effects">
+      {{!-- Effects Section --}}
+      {{> "systems/essence20/templates/item/parts/item-active-effects.hbs"}}
+    </div>
```
Comment (phildominguez) [inline templates/item/item-gear-sheet.hbs:34]:
```suggestion
      {{!-- Description Section --}}
      {{> "systems/essence20/templates/item/parts/item-description.hbs"}}
    </div>
    
    <div class="tab effects flexcol" data-group="primary" data-tab="effects">
      {{!-- Effects Section --}}
      {{> "systems/essence20/templates/item/parts/item-active-effects.hbs"}}
    </div>
```
Just a newline before effects tab

File: module/helpers/config.mjs [hunk 4]
```diff
@@ -311,6 +311,20 @@ E20.perkType = {
   role: "E20.PerkRole"
 };
 
+// Perk types
```
Comment (phildominguez) [inline module/helpers/config.mjs:314]:
```suggestion
// Gear types
```

**Review by phildominguez — CHANGES_REQUESTED** [review]

File: lang/en.json [hunk 5]
```diff
@@ -149,11 +149,17 @@
...
+    "GearComputers": "Computers & Electronics",
+    "GearExploration": "Exploration Gear",
+    "GearHazard": "Hazard Gear",
+    "GearKits": "Kits",
+    "GearMedical": "Medical Gear",
+    "GearMilitary": "Military Gear",
+    "GearSecurity": "Security Gear",
+    "GearSupport": "Support Gear",
+    "GearTools": "Tools",
+    "GearType": "Type",
```
[the remark this answers is not in the document]
Comment (JFranklin3000) [reply lang/en.json:162]: image

I went through all 4 books to get those types. 
I did add other last night but of course forgot to push it before bed. 

I am going back and forth on the Gear piece. I could definitely see removing it.

File: template.json (hunk 1 above)
Comment (JFranklin3000) [reply template.json:450]: Should quantity be on the item or the actor? I agree we should have a way to track quantity.

File: lang/en.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (phildominguez) [reply lang/en.json]: Ah okay, if they have that in the official names I'm cool with it

File: template.json [hunk 6]
```diff
@@ -441,14 +441,9 @@
...
-      "effectArea": null,
-      "effectShape": null,
-      "lightRange": {
-        "bright": null,
-        "dim": null
-      },
       "quantity": 1,
-      "toughness": null
+      "type": ""
+
```
Comment (phildominguez) [inline template.json:446]:
```suggestion
```

File: templates/item/item-gear-sheet.hbs (hunk 3 above)
Comment (phildominguez) [reply templates/item/item-gear-sheet.hbs:41]: You added the newline and then removed it 😄

**Review by phildominguez — CHANGES_REQUESTED** [review]

File: templates/item/item-gear-sheet.hbs [hunk 7]
```diff
@@ -3,45 +3,42 @@
...
-    {{#*inline "item-field-inputs"}}
-    <input type="number" name="system.lightRange.bright" value="{{system.lightRange.bright}}" placeholder={{localize "E20.LightRangeBright"}} />
-    <input type="number" name="system.lightRange.dim" value="{{system.lightRange.dim}}" placeholder={{localize "E20.LightRangeDim"}} />
-    {{/inline}}
-    {{/"systems/essence20/templates/item/parts/item-sheet-field.hbs"}}
+      {{#> "systems/essence20/templates/item/parts/item-sheet-field.hbs" label='E20.GearQuantity'}}
+      {{#*inline "item-field-inputs"}}
+      <input type="number" name="system.quantity" value="{{system.quantity}}" />
+      {{/inline}}
+      {{/"systems/essence20/templates/item/parts/item-sheet-field.hbs"}}
```
Comment (phildominguez) [inline templates/item/item-gear-sheet.hbs:33]:
```suggestion
```

File: templates/item/item-gear-sheet.hbs [hunk 8]
```diff
@@ -3,45 +3,42 @@
   {{!-- Item Sheet Header --}}
   {{> "systems/essence20/templates/item/parts/item-header.hbs"}}
 
+{{!-- Tabs --}}
```
Comment (phildominguez) [inline templates/item/item-gear-sheet.hbs:6]:
```suggestion
  {{!-- Tabs --}}
```
Haha I went too far

**Review by phildominguez — APPROVED** [review]
⚙️

**Pull request merged**